src/content-script/in-dapp-notification/index.ts

```diff
@@ -0,0 +1,120 @@
+import type { InDappNotification } from 'src/shared/types/InDappNotification';
+import * as styles from './styles.module.css';
+
+function preloadImage(url: string): Promise<HTMLImageElement> {
+  return new Promise((resolve, reject) => {
+    const img = new Image();
+    img.src = url;
+    img.onload = () => resolve(img);
+    img.onerror = reject;
+  });
+}
+
+const notifications = {
+  async chainChanged(networkName: string, networkUrl: string) {
+    const el = document.createElement('div');
+    el.className = `${styles.notification} ${styles.chainChanged}`;
+
+    let isIconLoaded = false;
+    try {
+      await preloadImage(networkUrl);
```

Comment:
This await impacts css animation start time which leads to small lag if icon wasn't preloaded. We need to take this into account for writing timeouts

Reply:
we had 2 options:
1) set `display: none` to the notification element before removing injected CSS
2) not remove injected CSS at all

I prefer the second one: zero effort, less lines of code